src/main/java/com/project/bookstore/controller/BookController.java

```diff
@@ -0,0 +1,55 @@
+package com.project.bookstore.controller;
+
+import com.project.bookstore.model.dto.BookDto;
+import com.project.bookstore.model.dto.CreateBookRequestDto;
+import com.project.bookstore.model.dto.UpdateBookRequestDto;
+import com.project.bookstore.service.BookService;
+import java.util.List;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/books")
+public class BookController {
+
+    private final BookService bookService;
+
+    public BookController(BookService bookService) {
+        this.bookService = bookService;
+    }
+
+    @GetMapping
+    public List<BookDto> getAll() {
+        return bookService.findAll();
+    }
+
+    @GetMapping("/{id}")
+    public BookDto getBookById(@PathVariable Long id) {
+        return bookService.getBookById(id);
+    }
```

Comment:
Only new code should be in this PR. See the task:
![image](https://github.com/roman680/Online-book-store/assets/20326858/56ef4938-3fb8-450f-9292-4d27554a4977)